draft-ietf-babel-information-model.md

```diff
@@ -158,6 +164,10 @@ string
 : A type representing a human-readable string consisting of a (possibly restricted)
   subset of Unicode and ISO/IEC 10646 {{ISO.10646}} characters.
 
+uint
+: A type representing an unsigned integer number. This information
+  model does not define a precision.
+

```

Comment:
Ahh! I see what you are doing here. Rather than define (with precision) what each attribute precision is, you are generally defining them as uint, and then making a note of the size in the comment section.